test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -5711,5 +5711,220 @@ public void Dotnet_New_Template_Restore_Pack_Success(string template)
                 Assert.True(File.Exists(nupkgPath));
             }
         }
+
+        [Theory]
+        [InlineData("netstandard2.1")]
+        [InlineData("net5.0")]
+        public async Task PackCommand_PackProject_PackageReference_PreReleaseDependency_Fails(string platform)
```

Comment:
Do you expect this test might behave differently based on what target framework the project uses?  I don't see the benefit of running this (I can't imagine a scenario where only one of the two catch a regression), or the other test below, multiple times, for different target frameworks.

Reply:
Sure, removed additional target frameworks.